### PR TITLE
Update deps. Improve ziti-fabric cli for terminators and links

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: etc/edge.router.tmpl.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+v: 3
```
Comment (andrewpmartinez) [inline etc/edge.router.tmpl.yml:1]: Interesting. I have a number of scripts that make use of template files just like this - just outside of the repo. 

Would it make sense to:
- move this to `etc/tmpl` as to not clutter this directory
- parameterize most settings

?

Comment (plorenz) [reply etc/edge.router.tmpl.yml:1]: sure. The controller host/ip could be templated. Anything else you'd want to templatize?

Comment (michaelquigley) [reply etc/edge.router.tmpl.yml:1]: We really need some sort of namespaced+`FABLAB_TEMPLATE_PATH'd template file resolution. If we had a `FABLAB_TEMPLATE_PATH` variable with `:` separated entries, people could put that stuff wherever they want.

Comment (michaelquigley) [reply etc/edge.router.tmpl.yml:1]: Wrong project. Disregard. 💯

Comment (plorenz) [reply etc/edge.router.tmpl.yml]: Moved and templated controller host/port

File: ziti-fabric/subcmd/list_links.go [hunk 2]
```diff
@@ -58,22 +58,24 @@ func listLinks(cmd *cobra.Command, args []string) {
...
+						status := "up"
 						if l.Down {
-							down = "->down<-"
+							status = "down"
 						}
 						// Convert nanoseconds to fractional seconds
-						srcLatency := float64(l.SrcLatency) / 1000000000.0
-						dstLatency := float64(l.DstLatency) / 1000000000.0
-						out += fmt.Sprintf("%-6s | %-24s -> %-24s | %-12s | %-4d | %-0.4f %-0.4f %s\n", l.Id, l.Src, l.Dst, l.State, l.Cost, srcLatency, dstLatency, down)
+						srcLatency := float64(l.SrcLatency) / 1_000_000_000.0
```
Comment (dovholuknf) [inline ziti-fabric/subcmd/list_links.go:74]: i still like constants over magic numbers but maybe that's just me? :)

**Review by dovholuknf — APPROVED** [review]

File: ziti-fabric/subcmd/list_links.go (hunk 2 above)
Comment (plorenz) [reply ziti-fabric/subcmd/list_links.go:74]: I'm not sure `float64(l.SrcLatency) / OneBillion` would provide any additional clarity :)

`float64(l.SrcLatency) / NanosecondsToSecondsScaleFactor` might, but I think the comment directly above does the same

**Pull request merged**